src/components/TextField/components/Resizer/tests/Resizer.test.tsx

```diff
@@ -0,0 +1,112 @@
+    contents: 'Contents',
+  };
+
+  describe('contents', () => {
+    it('renders contents', () => {
+      const contents = 'Contents';
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} contents={contents} />,
+      );
+      const contentsNode = findByTestID(resizer, 'ContentsNode');
+      const expectedHTML = '<div testid="ContentsNode">Contents<br></div>';
+      expect(contentsNode.html()).toBe(expectedHTML);
+    });
+
+    it('properly encodes HTML entities', () => {
+      const contents = `<div>&\nContents</div>`;
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} contents={contents} />,
+      );
+      const contentsNode = findByTestID(resizer, 'ContentsNode');
+      const expectedEncodedContents =
+        '&lt;div&gt;&amp;<br>Contents&lt;/div&gt;<br></div>';
+      expect(contentsNode.html()).toContain(expectedEncodedContents);
+    });
+  });
+
+  describe('minimumLines', () => {
+    it('renders a number of <br> tags equivalent to minimumLines', () => {
+      const minimumLines = 3;
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} minimumLines={minimumLines} />,
+      );
+      const breakingSpaces = findByTestID(resizer, 'MinimumLines');
+      expect(breakingSpaces.html()).toContain('<br><br><br>');
+    });
+  });
+
+  describe('onHeightChange()', () => {
+    it('is called on componentDidMount() if minimumLines is provided', () => {
+      const spy = jest.fn();
+      const minimumLines = 3;
+      mountWithAppProvider(
+        <Resizer
+          {...mockProps}
+          currentHeight={50}
+          onHeightChange={spy}
+          minimumLines={minimumLines}
+        />,
+      );
+      expect(spy).toHaveBeenCalledWith(0);
+    });
+
+    it('is not called if minimumLines is not provided', () => {
+      const spy = jest.fn();
+      mountWithAppProvider(
+        <Resizer {...mockProps} currentHeight={50} onHeightChange={spy} />,
+      );
+      expect(spy).not.toHaveBeenCalled();
+    });
+
+    it('is not called if currentHeight is the same as DOM height', () => {
+      const spy = jest.fn();
+      const currentHeight = 0;
+      mountWithAppProvider(
+        <Resizer
+          {...mockProps}
+          currentHeight={currentHeight}
+          onHeightChange={spy}
+          minimumLines={3}
+        />,
+      );
+      expect(spy).not.toHaveBeenCalled();
+    });
+
+    it('is called again on resize', () => {
+      const spy = jest.fn();
+      const currentHeight = 0;
+      const minimumLines = 3;
+      const resizer = mountWithAppProvider(
+        <Resizer
+          {...mockProps}
+          currentHeight={currentHeight}
+          onHeightChange={spy}
+          minimumLines={minimumLines}
+        />,
+      );
+      resizer.setProps({currentHeight: 1});
+      trigger(resizer.find(EventListener), 'handler');
+      expect(spy).toHaveBeenCalledWith(0);
+    });
+  });
+
+  describe('aria-hidden', () => {
```

Comment:
Do we need to test this? On one hand you probably wanna make sure this happens for a11y purposes on the other hand though I don't this you wanna test this here as its more of a linting thing.

Reply:
I've had this discussion in other test suites. One of the purposes of writing tests is to make sure that code changes don't break or remove things that should be there. I like testing aria attributes because it ensures the correct ones are there and they serve an important purpose. ?